test-data/unit/check-inference.test

```diff
@@ -430,19 +431,6 @@ def f() -> T: pass
 def g(a: T) -> None: pass
 [out]
 
-[case testUnsolvableInferenceResult]
```

Comment:
Consider enabling this test case and renaming it if the output doesn't correspond to the original intent of the test case.